### PR TITLE
doc: fox uwsgi example

### PR DESCRIPTION
**Pull request by jd**
doc: fox uwsgi example

### CODE REVIEW

File: doc/source/running.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ machines.
 The following uwsgi configuration file can be used::
 
   [uwsgi]
-  http = localhost:8041
```
Comment (chungg) [inline doc/source/running.rst]: this is actually valid... it's what i have in my deployment.

Comment (jd) [inline doc/source/running.rst]: what version of uwsgi do you have?
got 2.0.15 here

Comment (chungg) [inline doc/source/running.rst]: 2.0.14, according to docs, http makes uwsgi act similar to nginx/apache? http://uwsgi-docs.readthedocs.io/en/latest/ThingsToKnow.html

Comment (jd) [inline doc/source/running.rst]: Ok, so I guess it's a compile option missing on my install or something. I'll abandon.

**Pull request closed without merging**